src/Microsoft.EntityFrameworkCore.Tools.DotNet/Microsoft.EntityFrameworkCore.Tools.DotNet.csproj

```diff
@@ -7,9 +7,9 @@
     <OutputType>Exe</OutputType>
     <IsPackable>false</IsPackable>
     <RootNamespace>Microsoft.EntityFrameworkCore.Tools</RootNamespace>
+    <NetCoreAppImplicitPackageVersion>1.0.0</NetCoreAppImplicitPackageVersion>
```

Comment:
Seems weird to override a property with *implicit* in the name... But I guess it's the "version of the implicit package", not the "implicit version of the package".